[WIP] replace kbnTimepicker directive with EuiSuperDatePicker

nreese opened the pull request:
## Summary

Summarize your PR. If it involves visual changes include a screenshot or gif.

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.

- [ ] This was checked for cross-browser compatibility, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [ ] [Documentation](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#writing-documentation) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios
- [ ] This was checked for [keyboard-only and screenreader accessibility](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility#Accessibility_testing_checklist)

### For maintainers

- [ ] This was checked for breaking API changes and was [labeled appropriately](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#release-notes-process)
- [ ] This includes a feature addition or change that requires a release note and was [labeled appropriately](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#release-notes-process)

nreese committed:
replace kbnTimepicker directive with EuiSuperDatePicker

nreese committed:
remove kbnTimepicker directive

nreese committed:
remove bootstrap datepicker

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/5248/)

loixlab committed:
[CCR] Tell user when multiple auto-follow patterns try to replicate the same data (#27783)

* Fix small tipo

* [CCR] Add API to fetch ccr stats

* Add deserialization for recentAutoFollowErrors & autoFollowedClusters

* Add UI to display recent auto-follow pattern errors

sorenlouv committed:
[APM] StickyProperties should render components as values (#27814)

sorenlouv committed:
Revert changes to Monitoring accidentally introduced in #27591 (#27830)

pickypg committed:
Remove Unnecessary Delay (#25897)

This adds logic so that the code will wait until any previous
attempt to send telemetry causes follow-up attempts to wait until
it has completed sending (in the unlikely event that it takes
longer than a minute, we don't want to waste time / resources).

Commit:
[interpreter/loadPlugins] avoid deleting globals added by plugins (#27171)

Plugins loaded by the `@kbn/interpreter` can sometimes setup global values while loading, like the regeneratorRuntime for instance, but the current plugin loading code is deleting every global that was added during plugin load. This changes the logic to only cleanup the `canvas` global after loading the canvas plugins.

resolves https://github.com/elastic/kibana/issues/27162

cqliu1 committed:
Added checks to mousemove handler to not commit all mouse movements (#27818)

nreese committed:
fix console documenation (#27862)

nreese committed:
[Maps] refactor replaceLayerList to use addLayer and removeLayer (#27823)

nreese committed:
[Maps] trigger digest cycle on kbnUrl.change (#27812)

* [Maps] trigger digest cycle on kbnUrl.change

* remove extra space

dimitris-athanasiou committed:
[ML] Remove use of types in datafeeds  (#27572)

This commit removes the use of types in datafeeds
and a few more places where it was possible.

Relates #26968

mattkime committed:
NOTICE (#27875)

kindsun committed:
If dynamic vector value is null, set color to transparent (#27821)

nreese committed:
[Maps] support elasticsearch documents containing array of geometry values (#27846)

* [Maps] support elasticsearch documents containing array of geometry values

* clean up

* use reduce instead of map and flatten

* replace map and flatten with reduce in geoShapeToGeometry

nreese commented:
Closing. Moved work into a feature branch `feature/global-date-picker`
